pages/404.vue

```diff
@@ -2,6 +2,9 @@
     <div
         class="max-w-7xl mx-auto sm:px-6 lg:px-8 flex flex-1 flex-col justify-center"
     >
+        <Head>
```

Comment:
nit pick, all of our tags are lowercase. Mind changing them all to lowercase?

Reply:
fixed it